streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImpl.java

```diff
@@ -175,11 +171,7 @@ public StreamsMetricsImpl(final Metrics metrics,
     }
 
     private static Version parseBuiltInMetricsVersion(final String builtInMetricsVersion) {
```

Comment:
After removing `FROM_0100_TO_24`, I think we can remove this `parseBuiltInMetricsVersion`, too.

Reply:
Removed it.